src/base-element.js

```diff
@@ -261,10 +261,13 @@ export class BaseElement {
   /**
    * Subclasses can override this method to opt-out of rendering the element
    * when it is not currently visible.
-   * @return {boolean}
+   * Returning a boolean allows or prevents rendering outside the viewport at
+   * any distance, while returning a number allows rendering only when the
+   * element is within X viewports of the current viewport.
```

Comment:
Understand. But logically it's very easy to explain what 0 and 1 mean if we bring the reference point to the simplest case. We can say 0 means you are already in viewport. Or we can say that 1 means you are in the next viewport - whichever, as long as we tie in 0 or 1.